R/stacking.R

```diff
@@ -786,8 +789,7 @@ stackingSpecifiedByVariable <- function(manual.stacking,
             !isValueAttributesMergable(v.val.attr[removeNA(group.ind)]))
         {
             warning("The manual stacking input '", input.text,
-                    "' has been ignored as it contains variables with mismatching types or value attributes. ",
-                   "Every value must have identical labels in each variable it appears in.")
+                    "' has been ignored as it contains variables with mismatching types or value attributes.")
```

Comment:
Did you mean to remove the new addition to this comment?

Reply:
No. Thanks for catching that!